latexrun

```diff
@@ -1629,7 +1629,7 @@ class BibTeX(Task):
 
         if self.__is_biber():
             outbase = os.path.join(cwd, outbase)
-        outputs = [outbase + '.bbl', outbase + '.blg']
+        outputs = [outbase+ext for ext in ['.bbl','.blg','.bcf']
```

Comment:
You are missing a closing bracket

Reply:
Thanks for catching that! I've committed a revised version.